ui/Screen/Project/Commit.svelte

```diff
@@ -44,6 +44,232 @@
 
       // TODO(cloudhead): Fetch branch from backend.
       commit.branch = "master";
+      commit.changeset = {
```

Comment:
Is this data shaped on something? Maybe we can link it here if so.

Reply:
It's what I could come up with which makes the frontend easy to build with not too much logic. Unless there's a good reason not to, I would say that we transform whatever `surf` gives us into this format. It's also probably a good idea to discuss this with `surf` to nudge them in this direction, or have them convince us otherwise.